pages/networks/_meta.json

```diff
@@ -0,0 +1,4 @@
+{
+    "network1": "Testnet"
```

Comment:
Using generic `network1` so that relative links to `../network1/..` will not break